pkgs/development/python-modules/schema-salad/default.nix

```diff
@@ -56,6 +57,11 @@ buildPythonPackage rec {
     ++ cachecontrol.optional-dependencies.filecache
     ++ lib.optionals (pythonOlder "3.9") [ importlib-resources ];
 
+  patches = [ (fetchpatch {
+    url = "https://patch-diff.githubusercontent.com/raw/common-workflow-language/schema_salad/pull/840.patch";
```

Comment:
this is referencing a pull request instead of a commit, which can have it's contents changed.